src/Logging/Logging.Console/src/ConsoleLoggerProvider.cs

```diff
@@ -23,6 +23,7 @@ public ConsoleLoggerProvider(IOptionsMonitor<ConsoleLoggerOptions> options)
             _options = options;
             _loggers = new ConcurrentDictionary<string, ConsoleLogger>();
 
+            _optionsReloadToken = _options.OnChange(ReloadLoggerOptions);
```

Comment:
Could you move this to be below `ReloadLoggerOptions`?

Reply:
Yes. This is just to keep consistent with other logger providers(such as [Extensions/src/Logging/Logging.AzureAppServices/src/BatchingLoggerProvider.cs](https://github.com/aspnet/Extensions/blob/master/src/Logging/Logging.AzureAppServices/src/BatchingLoggerProvider.cs))